data/human/news.txt

```diff
@@ -62,7 +62,7 @@ news "merchant anywhere"
 			"For some reason, boring jobs almost never hang out in the spaceport. So, if you are looking for interesting or at least varied work..."
 			"If you can't stand the heat, stay out of the solar storms."
 			"Silence is golden, except when you are listening to the life support systems."
-			`Even in this day and age some people still say "the sun goes down", instead of the more appropriate "the sun goes behind the horizon".`
+			`Even in this day and age some people still say "the sun goes down," instead of the more appropriate "the sun goes behind the horizon."`
```

Comment:
```suggestion
			"Even in this day and age some people still say 'the sun goes down,' instead of the more appropriate "the sun goes behind the horizon.'"
```

Reply:
Fixed, including the "-character before the sun.